components/arena/python/arena/_arena_distributed_tf_op.py

```diff
@@ -26,10 +26,10 @@ def estimator_op(name, image, command,
                       gpus, rdma, 
                       tensorboard, 
                       worker_port, annotations=[],
-                      evaluator=False, evaluator_cpu_limit=0, evaluator_memory_limit=0, 
+                      evaluator=False, evaluator_cpu_limit='0', evaluator_memory_limit='0', 
                       env=[], data=[], sync_source=None,
                       metrics=['Train-accuracy:PERCENTAGE'],
-                      arena_image='cheyang/arena_launcher:v0.6',
+                      arena_image='cheyang/arena_launcher:v0.7',
```

Comment:
my pet peeve :-) Let`s try and move away from docker repos tied to an individual - doesn't give confidence over the maintainability of the project.

Reply:
Yeah, do you have any guidance?